File: iPad/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/json/QuoteSizeList.m
//
//  QuoteSizeList.m
//  IOSClientApiTester
//
//  Created by felix on 7/25/13.
//  Copyright (c) 2013 allone. All rights reserved.
//

#import "QuoteSizeList.h"
#import "JEDIDateTime.h"
#import "JEDISystem.h"

static NSString * gs_QSList_jsonId  =@"QuoteSizeList";
static NSString * gs_QSList_sizes   =@"1";


@implementation QuoteSizeList

//-------------------------------------------------------------
-(id)               init
{
    self = [super init];
    
    if(self != nil)
    {
        [super setEntry:@"jsonId" entry:gs_QSList_jsonId];
    }
    
    return self;
}

//-------------------------------------------------------------
-(NSMutableArray *) getSizes // QuoteSizeData[]
{
    JEDI_SYS_Try
    {
        return [super getEntryArray:gs_QSList_sizes];
    }
    JEDI_SYS_EndTry
    
    return nil;
}

//-------------------------------------------------------------
-(void)             setSizes:(NSMutableArray *) sizes
{
    JEDI_SYS_Try
    {
        [super setEntry:gs_QSList_sizes entry:sizes];
    }
    JEDI_SYS_EndTry
}

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/CSTS3/comm/jsondata/QuoteSizeList.java
package jedi.ex01.CSTS3.comm.jsondata;

import jedi.ex01.CSTS3.comm.struct.QuoteSizeData;


public class QuoteSizeList extends allone.json.AbstractJsonData {

	public static final String jsonId = "QuoteSizeList";

	public static final String sizes = "1";

	public QuoteSizeList(){
		super();
		setEntry("jsonId", jsonId);
	}

	public QuoteSizeData[] getSizes() {
		try {
			QuoteSizeData[] data=getEntryObjectVec(QuoteSizeList.sizes,new QuoteSizeData[0]);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setSizes(QuoteSizeData[] sizes) {
		setEntry(QuoteSizeList.sizes, sizes);
	}


}